ExampleApp.xcodeproj/ExampleApp.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,14 @@
+{
+  "pins" : [
+    {
+      "identity" : "omise-ios",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/omise/omise-ios.git",
+      "state" : {
+        "branch" : "feature/MIT-1520",
```

Comment:
That suppose to use current working branch (SPM doesn't have this option) so I suppose to change it during development. I will change it to master.

Reply:
This change will come in next PR or this PR?

Reply:
Yep, this is a weak part of SPM as I mentioned before. It's supposed to have a current branch name during development and specific (latest) version of SDK in production.

Pointing it to any other branch could cause more issues and reports as developer may not pay attention to the specified branch and get unexpected codebase.

Pointing to destination branch doesn't make it better as well because ExampleApp won't be working in current branch if anyone would like to run and test it.

Having a short live branch on other hand will force Xcode to show the error and force developer to change the version or branch: `could not find a branch named ‘feature/MIT-1520’ in https://github.com/omise/omise-ios.git`

I'm agreed that it's not the ideal solution and I'll try to make it better when I find the way. 

I'd just add that the destination branch is supposed to be tested anyway at some point and I'll be forced to change this value. When I'm going to bump the version and create a git tag for production I'll be also forced to change this value to the new branch to test it (in this case it would be specific version, not a branch name). 

So far as long as I'm running app to test after merging I'll be always forced to change this value and make probably make another mini PR. It's a first PR where I introduced this dedicated ExampleApp so will see how it goes and what I can do to make it better.

The End. 

P.S. Thanks for spending your time reading this 🤣